taggedTemplateSolution.js

```diff
@@ -0,0 +1,38 @@
+//With the template provided, write a template tag that will validate an SQL statement:
+//Only SELECT or UPDATE statements are valid - write validation logic which allows only SELECT, UPDATE
+//The passwords table cannot be altered
+//there is no order by clause, add it and order by asc
```

Comment:
Implement order by asc functionality

Reply:
It will order the data in ascending order only. It is a by default functionality of SQL.

Reply:
We shall build the query in JS so add it here as a if statement